website/docs/r/qldb_ledger.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Since we're likely going to merge https://github.com/terraform-providers/terraform-provider-aws/pull/10684 shortly, we'll need to add a `subcategory` to this documentation page:

```suggestion
---
subcategory: "Quantum Ledger Database (QLDB)"
```